### PR TITLE
Bonfire v3 (switch to namespace operator)

### PR DESCRIPTION
**Pull request by bsquizz**

### CODE REVIEW

File: cicd/bootstrap.sh [hunk 1]
```diff
@@ -43,7 +43,10 @@ python3 -m venv .bonfire_venv
 source .bonfire_venv/bin/activate
 
 pip install --upgrade pip 'setuptools<58' wheel
-pip install --upgrade 'crc-bonfire>=2.17.2'
+pip install --upgrade 'git+https://github.com/RedHatInsights/bonfire.git@use-ns-operator'
```
Comment (Jason-RH) [inline cicd/bootstrap.sh:46]: I'll add a note here that this is also temporary for testing

File: cicd/_common_deploy_logic.sh [hunk 2]
```diff
@@ -61,7 +61,7 @@ function collect_k8s_artifacts {
     get_oc_events
     oc get all -n $NAMESPACE -o yaml > $K8S_ARTIFACTS_DIR/oc_get_all.yaml
     oc get clowdapp -n $NAMESPACE -o yaml > $K8S_ARTIFACTS_DIR/oc_get_clowdapp.yaml
-    oc get clowdenvironment env-$NAMESPACE -o yaml > $K8S_ARTIFACTS_DIR/oc_get_clowdenvironment.yaml
+    oc get clowdenvironment $NAMESPACE -o yaml > $K8S_ARTIFACTS_DIR/oc_get_clowdenvironment.yaml
```
Comment (Jason-RH) [inline cicd/_common_deploy_logic.sh:64]: Need to add "env-" back here

File: bonfire/bonfire.py [hunk 3]
```diff
@@ -630,17 +544,18 @@ def _list_namespaces(available, mine, output):
             for ns in namespaces:
                 data[ns.name] = {
                     "reserved": ns.reserved,
-                    "ready": ns.ready,
-                    "requester": ns.requester_name,
+                    "status": ns.status,
```
Comment (bsquizz) [inline bonfire/bonfire.py:547]: FYI @skarekrow not sure if your iqe-tools checks this field in the JSON returned here from list namespaces ... you might need to alter the key name you're looking for when bonfire v3 is released

Comment (skarekrow) [reply bonfire/bonfire.py:547]: We check the `expires_in` field, but we do add the `ns.ready` field to a dict, so will note that. Thanks!

**Pull request merged**